src/components/alert/alert.tsx

```diff
@@ -398,7 +398,7 @@ export class Alert implements OpenCloseComponent, LoadableComponent, T9nComponen
   @State() queued = false;
 
   /** the close button element */
-  private closeButton?: HTMLButtonElement;
```

Comment:
This is fine for now, but for the future, let's keep these cleanup changes separate unless the code specific PR changes require a refactor or if it's in the modified line vicinity. Reasoning: there are no additional changes here that relevant to the PR.